client/src/components/Register/style.scss

```diff
@@ -0,0 +1,260 @@
+.container-fluid {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0;
+  height: 1024px;
+  background: $white;
+
+
+  .bg-img {
+    top: 0;
+    left: 0;
+    width: 640px;
+    height: 1024px;
+    z-index: 1;
+    background: url('../../assets/img/signup-bg.jpg');
+
+    .logo-in-sign-up {
+      width: 253px;
+      height: 83px;
+      z-index: 2;
+      text-align: center;
+      background: $crimson;
+
+      h1 {
+        margin: 0;
+        font-family: Open Sans, serif;
+        font-size: 1.5rem;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 83px;
+        color: $rose-white;
+      }
+    }
+  }
+
+  .create-account-form {
+    right: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    width: 800px;
+    height: 1024px;
+
+    .log-in {
+      top: 0;
+      right: 0;
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      width: 375px;
+      height: 83px;
+      margin: 0 0 0 45%;
```

Comment:
nope. it is a very bad idea. Why margin is soooo big?

Reply:
fixed